content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -239,8 +241,8 @@ export class _DiscoveryStreamBase extends React.PureComponent {
           isFixed={true}
           learnMore={{
             link: {
-              href: message.header.link_url,
-              id: message.header.link_text,
+              href: topStories.learnMore.link.href,
+              id: topStories.learnMore.link.id,
```

Comment:
This is changing the behavior where learnMore link and text could be set by the server. I don't think this is desired. E.g., a few lines lower the title text is from the server's message response.